### DISCUSSION TITLE
Add this features please

### DISCUSSION THREAD

**Discussion by DinhHuy2010** [ThioJoe/YT-Spammer-Purge#255] [Ideas]
Do you can add features to report someone else video with non-support fillers mode?

**Comment by ThioJoe** [comment]
> Do you can add features to report someone else video with non-support fillers mode?

Can you re-state the question?

**Reply by DinhHuy2010** [reply]
Could you add the the options to report someone else video with non-support fillers mode?

**Reply by ThioJoe** [reply]
I have no idea what "non-support fillers" means

**Reply by DinhHuy2010** [reply]
this one
![Screenshot 2021-12-27 123841](https://user-images.githubusercontent.com/95196459/147673704-4dfc3ba3-911d-417f-8fda-f427b842fd75.png)

**Reply by ThioJoe** [reply]
Ah I see. Yea when scanning someone elses channel it only allows the auto smart mode to prevent abuse

**Reply by DinhHuy2010** [reply]
ok (I guess issue for youtube api)
